src/controllers/list.js

```diff
@@ -771,7 +772,7 @@ define(['globalize', 'listView', 'layoutManager', 'userSettings', 'focusManager'
                     loading.hide();
 
                     if (refresh) {
-                        focusManager.autoFocus(self.itemsContainer);
+                        focusManager.default.autoFocus(self.itemsContainer);
```

Comment:
```suggestion
                        focusManager.autoFocus(self.itemsContainer);
```

Since you have got  `    focusManager = focusManager.default || focusManager;` at the start of the file, the .default isn't needed